Make Travis-CI Great Again

Current Travis output:

```
  1297 passing (372ms)
  14 failing

  1) top_sites #74 Google Admin should have a valid url:
     AssertionError: expected http:/admin.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  2) top_sites #75 Calendar should have a valid url:
     AssertionError: expected http:/calendar.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  3) top_sites #76 Contacts should have a valid url:
     AssertionError: expected http:/contacts.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  4) top_sites #77 Docs should have a valid url:
     AssertionError: expected http:/docs.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  5) top_sites #78 Drive should have a valid url:
     AssertionError: expected http:/drive.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  6) top_sites #79 Forms should have a valid url:
     AssertionError: expected http:/forms.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  7) top_sites #80 Gmail should have a valid url:
     AssertionError: expected http:/mail.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  8) top_sites #81 Groups should have a valid url:
     AssertionError: expected http:/groups.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  9) top_sites #82 Hangouts should have a valid url:
     AssertionError: expected http:/hangouts.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  10) top_sites #83 Google Plus should have a valid url:
     AssertionError: expected http:/plus.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  11) top_sites #84 Sheets should have a valid url:
     AssertionError: expected http:/sheets.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  12) top_sites #85 Sites should have a valid url:
     AssertionError: expected http:/sites.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  13) top_sites #86 Slides should have a valid url:
     AssertionError: expected http:/slides.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)
  14) top_sites #87 Photos should have a valid url:
     AssertionError: expected http:/photos.google.com/ to be a valid url: expected false to be true
      at Function.assert.isTrue (node_modules/chai/lib/chai/interface/assert.js:332:31)
      at Context.<anonymous> (test/index.test.js:31:16)

npm ERR! Test failed.  See above for more details.
```

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 # Tippy Top Sites
 
-[![Build Status](https://travis-ci.org/nchapman/tippy-top-sites.svg?branch=master)](https://travis-ci.org/nchapman/tippy-top-sites)
+[![Build Status](https://travis-ci.org/mozilla/tippy-top-sites.svg?branch=master)](https://travis-ci.org/mozilla/tippy-top-sites)
 
 This repo is meant to provide a hand curated set of icons, colors, and titles for use in [Activity Stream](https://github.com/mozilla/activity-streams). It is meant as a temporary solution to ugly data. We should do something much better very soon.
 

